lib/services/iotaRedirector.js

```diff
@@ -209,7 +216,7 @@ function processRequests(req, res, next) {
 
             if (statusCodes.length === 1) {
                 if (results.length === 1) {
-                    res.status(statusCodes[0]).json(results[0]);
+                    res.status(statusCodes[0]).json(results[0][1]);
```

Comment:
I am. As you can see, the results attribute is a result from a `map()` function (that is a vector) whose elements are the result of the `sendRequest()' function (always vectors, as you can see in line 203).